.github/workflows/articles_branch.yaml

```
@@ -95,10 +95,14 @@ jobs:
             owner="${BASH_REMATCH[1]}"
             repo="${BASH_REMATCH[2]}"
             artifact_id="${BASH_REMATCH[3]}"
-            preview_link="（[プレビュー](https://word-coins.github.io/artifact-previewer/?owner=${owner}&repo=${repo}&artifact_id=${artifact_id})）"
+            job_url="https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+            preview_link="（[プレビュー](https://word-coins.github.io/artifact-previewer/?owner=${owner}&repo=${repo}&artifact_id=${artifact_id})）"$'\n\n'
+            expired="有効期限(おおよそ): $(date --date "7 days" +%F)"$'\n'
+            renew="有効期限が切れた場合は[こちら](${job_url})からRe-run all jobsを実行してください"
```

```suggestion
            renew="有効期限が切れた場合は[こちら](${job_url})から<kbd>Re-run all jobs</kbd>を実行してください"
```

カギカッコだったり `<kbd>` で囲むと読みやすくなるように思った